### DISCUSSION TITLE
Cant install or de-install the bluetooth

### DISCUSSION THREAD

**Discussion by MastahKilla** [nefarius/DsHidMini#324] [Q&A]
Hello guys im new here before this post i try to read everything on discussion section about this but nothing  helped.

Last year i used this softwere without any problem with bluetooth and usb cabel this year something happend and now bluetooth is not working ,but says i have already installed version but is not working.

I try to re-install everything but without luck.
Now i cant install the new bluetooth version bacause its says: i have to un-install the previous version first but how to delete the previous version of this driver and install the new one?
![Екранна снимка (26)](https://github.com/nefarius/DsHidMini/assets/147328683/4c00c59b-ce53-4808-b099-454b0a1b264f)

Thanks in advance!

**Comment by nefarius** [comment]
The FAQ covers how to solve this exact case.
